src/index.js

```diff
@@ -41,7 +55,7 @@ export default function (h) {
       var parsed = parse(decls);
       return function (props, children) {
         var classes = [props && props.class, parsed].filter(Boolean)
-        return h(tag, Object.assign({}, props, { class: classes.join(' ') }), children)
+        return h(tag, merge(props, { class: classes.join(' ') }), children)
```

Comment:
@morishitter @maxholman Sorry, but wouldn't be easier to just mutate the object instead of merging in this particular case? Because picostyle is trying to be super small & everything. 😄

Reply:
It would certainly keep the file size smaller, it's just a trade off between byte size and what you might arguably call "bad behaviour" by mutating the object.

I'm not 100% sure what all the use cases are, so I cant form a real opinion on how "bad" it might be.

Some stats to assist:

Original: 804 bytes built (496 bytes gzip)
Object mutate: 851 bytes built (513 bytes gzip)
Merge fn: 938 bytes built (548 bytes gzip)

Reply:
There's also the "bad feeling" of we are just copy & pasting merge from hyperapp. But arguing over that would lead to a different rabbit hole (e.g., should hyperapp export its own merge definition?, etc.)

Quick question: what if the props object already has a `class` property?

Reply:
Any existing `class` is handled in this line:

```javascript
var classes = [props && props.class, parsed].filter(Boolean)
```

Actually, I can do this a different way and save another 20 bytes/10bytes gzip :)

Reply:
@maxholman I say we go for mutating the props object directly. The entire library is not particularly written in a functional way and one of the main concern is to save bytes, so it's okay. My two cents. 👋